Usage Console App (#4180)
SoftwareAG/c8y-docs: content/console/account-bundle/overview.md

---
weight: 10
title: Overview
layout: redirect
sector:
  - platform_administration
---

> The Overview page gives a high level summary of the consumption for each environment. 

Reviewer: ```suggestion
The **Overview** page provides a high-level summary of the consumption for each environment. 
```

> The Overview page gives a high level summary of the consumption for each environment. 

Reviewer: It is confusing that the Console application has 3 Overview pages. At least you should mention here to which one this text refers.

Author: Addressed in https://github.com/Cumulocity-IoT/c8y-docs/pull/4180/commits/e2830e034c869d04c61eb8cd3a9643c4d284cdec

> The **Account Overview** page provides a high-level summary of the consumption for each environment. 

Reviewer: ```suggestion
The **Account overview** page provides a high-level summary of the consumption for each environment. 
```

Reviewer: This is actually an error in the localization.

Author: I will get this changed in the UI.

<img src="/images/console/account/overview/overview.png" alt="Account Overview">

> Much of the data is the same as the Environments page, but added is consumption and usage data. For every environment, the two core metrics (Messages and Operational Data Store) are present, showing usage over the past months. In addition, contractual information is present in the upper right hand corner, showing important details about your contract with {{< company-c8y >}}. 

Reviewer: ```suggestion
The data is similar to the data shown in the **Environments** page, but consumption and usage data are added. For every environment, the two core metrics (Messages and Operational Data Store) are present, showing usage over the past months. In addition, contractual information is displayed at the top right, showing important details about your contract with {{< company-c8y >}}. 
```

> The data is similar to the data shown in the **Environments** page, but consumption and usage data are added. For every environment, the two core metrics (Messages and Operational Data Store) are present, showing usage over the past months. In addition, contractual information is displayed at the top right, showing important details about your contract with {{< company-c8y >}}. 

Reviewer: ```suggestion
The data is similar to the data shown in the **Environments** page, but consumption and usage data are added. For every environment, the two core metrics (messages and Operational Data Store) are present, showing usage over the past months. In addition, contractual information is displayed at the top right, showing important details about your contract with {{< company-c8y >}}. 
```
